### PR TITLE
Console Buffering

### PR DESCRIPTION
**Pull request by MikeInnes**
ref #5

### CODE REVIEW

File: lib/console/console.coffee [hunk 1]
```diff
@@ -44,11 +44,26 @@ class Console
...
-  info: (s) -> @view.add @view.infoView(s), @isInput
+  @debounce: (t, f) ->
+    timeout = null
+    (args...) ->
+      if timeout? then clearTimeout timeout
+      timeout = setTimeout (=> f.call this, args...), t
+
+  @buffer: (f) ->
+    buffer = []
+    flush = @debounce 10, ->
```
Comment (sglyon) [inline lib/console/console.coffee:55]: Am I reading this right? Does the 10 mean you wait 10 ms before displaying the messages on the console?

Comment (MikeInnes) [inline lib/console/console.coffee:55]: Yup, that's exactly right. It's a pretty arbitrary choice but it seems to capture single `println`s roughly right.

Comment (sglyon) [inline lib/console/console.coffee:55]: Great, looks like a strict improvement to me

Comment (MikeInnes) [inline lib/console/console.coffee:55]: Cool, thanks for taking a look

**Pull request merged**